add-ons/dspace-docker-tools/ingestAIP.sh

```diff
@@ -3,10 +3,14 @@
 CHECKFILE=/dspace/assetstore/ingest.hasrun.flag
 
 export JAVA_OPTS="${JAVA_OPTS} -Xmx2500m -Dupload.temp.dir=/dspace/upload -Djava.io.tmpdir=/tmp"
+echo ${LOCALCFG1} >> /dspace/config/local.cfg
```

Comment:
See my other comment

Reply:
@tdonohue , thanks for taking a look at this!  

The 3 variables would provide 3 different override files with a slot to insert their own overrides.  I initially planned for the Oracle override file to use one of these slots, but then I encountered the value concatenation issue we discussed yesterday.

This highlights how nice it would be if we could simply pass these in through the environment.  See ttps://stackoverflow.com/questions/54545571/passing-variable-aaa-bbb-into-a-docker-container-process/54581727#54581727 for a history on that.

@pnbecker also had questions about this.  I wonder if we could program the apache config class to translate a key like "db_url" to "db.url".  If so, we could simply use environment vars to pass in overrides.

Reply:
I posted a question related to this at https://stackoverflow.com/questions/55128269/apache-config-set-property-names-containing-a-period-via-environment-variables

Reply:
Resolving.  See the discussion in https://github.com/DSpace-Labs/DSpace-Docker-Images/pull/93 for an alternative approach